label_studio/ml/models.py

```diff
@@ -391,6 +529,16 @@ def _validate_ml_api_result(ml_api_result, tasks, curr_logger):
     return True
 
 
+@receiver(pre_delete, sender=MLBackend)
+def modify_project_model_version(sender, instance, **kwargs):
+    """ """
+    project = instance.project
+
+    if project.model_version == instance.title:
+        project.model_version = None
+        project.save()
```

Comment:
please use update_fields to avoid the potential for clobbering other updates to `project`. like `project.save(update_fields=['model_version'])`